pkg/names/names.go

```diff
@@ -24,8 +24,13 @@ import (
 )
 
 const (
-	ServerContainer      = "server"
-	ServerContainerIndex = 0
+	ServerContainer = "server"
+	NMAContainer    = "nma"
+)
+
+const (
+	ServerContainerIndex = iota
```

Comment:
I guess if we set the nma as the first container, we will need to pick ServerContainerIndex based on the deployment.

Reply:
Yes, but given that nma is sidecar will be the permanent solution, it make sense to have it as 1st container.

Reply:
Yes I mean I can add a logic that picks the correct index based on the deployment type.